### DISCUSSION TITLE
〔 Comments Section 〕Fallout Shelter Online: Dwellers

### DISCUSSION THREAD

**Discussion by morgosus** [morgosus/morgosus.github.io#12] [Comments]
A comments section for the [Fallout Shelter Online: Dwellers](https://www.toms.click/gaming/foso/2021/08/13/fallout-shelter-dwellers.html) post.

**Comment by Emptycz** [comment]
Which dwellers are the best?

**Reply by morgosus** [reply]
While there are no set "best" ones and you can always work with what you have, my personal favorites are Desdemona (whom you can get in the combat zone, she's awesome), Amata (The best regular healer imo), Strong (warning, leaves dwellers exposed) / Clint / Cait, Nick, Neriah, Maykr and Marauder. For GvE I'd suggest Piper, Dave (equipped with the stealth armor) and pretty much any strong damage dealers. You can use Roach King literally everywhere without even increasing his stars. Finally, when you need to clear Rads, I'd suggest Neriah (best radiation healer) or Madison.

Oh, and stay away from Kellog. He's not good.
